cmd/kubermatic-api/swagger.json

```diff
@@ -14212,6 +14222,54 @@
       },
       "x-go-package": "k8c.io/kubermatic/v2/pkg/crd/kubermatic/v1"
     },
+    "PodDNSConfig": {
+      "description": "PodDNSConfig defines the DNS parameters of a pod in addition to\nthose generated from DNSPolicy.",
+      "type": "object",
+      "properties": {
+        "nameservers": {
+          "description": "A list of DNS name server IP addresses.\nThis will be appended to the base nameservers generated from DNSPolicy.\nDuplicated nameservers will be removed.\n+optional",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "x-go-name": "Nameservers"
+        },
+        "options": {
+          "description": "A list of DNS resolver options.\nThis will be merged with the base options generated from DNSPolicy.\nDuplicated entries will be removed. Resolution options given in Options\nwill override those that appear in the base DNSPolicy.\n+optional",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PodDNSConfigOption"
+          },
+          "x-go-name": "Options"
+        },
+        "searches": {
+          "description": "A list of DNS search domains for host-name lookup.\nThis will be appended to the base search paths generated from DNSPolicy.\nDuplicated search paths will be removed.\n+optional",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "x-go-name": "Searches"
```

Comment:
Wouldn't be "Domains" more adapted/clear than searches? Or "Search domains" for the sake of clarity.

Reply:
Just saw K8s is using this, so forget about my comment.